bin/GameIndex.dbf

```diff
@@ -5563,6 +5563,19 @@ Serial = SCUS-97396
 Name   = World Tour Soccer 2005 [Demo]
 Region = NTSC-U
 ---------------------------------------------
+Serial = SCUS-97397
+Name   = Syphon Filter - The Omega Strain Public Beta 1
+Region = NTSC-U
+EETimingHack = 1 // Fixes random hangs.
+[patches = C909A32E]
+	author=Refraction & YukiXXL
+	//COP2 fix.
+	patch=0,EE,00367798,word,48438000
+	patch=0,EE,0036779C,word,4B06521B
+	//Offline button fix
```

Comment:
Maybe format the comments better so they match the rest of the file.

Reply:
Huh?

Reply:
The one existing example I looked up had a space after `//` and before the actual comment.
```
// COP2 fix.
```

Reply:
Well, "COP2 fix" do not explain much what is going on here, I think you should write more about why does it need a patch, don't hesitate to contact the team as someone will probably know that.